tests/figures/chart/charts_component.test.ts

```diff
@@ -1165,6 +1182,109 @@ describe("charts", () => {
     });
   });
 
+  describe("Cell reference for title", () => {
```

Comment:
The tests that need a mounted component should probably be in chart_title.test.ts (that should be renamed to `chart_title_component.test.ts` as per our test naming convention), and the one that only need the model should not mount a spreadsheet, and be either in `chart_plugin.test.ts` or maybe in a new file `chart_title_plugin.test.ts`